src/test/run-pass/regions/regions-lub-ref-ref-rc.rs

```diff
@@ -11,12 +11,12 @@
 use std::rc::Rc;
 
 #[derive(Clone)]
-enum CachedMir<'mir> {
+enum CachedBody<'mir> {
     Ref(&'mir String),
     Owned(Rc<String>),
 }
 
-impl<'mir> CachedMir<'mir> {
+impl<'mir> CachedBody<'mir> {
```

Comment:
No need to change this test - alternatively, `CachedMir` can be renamed to `Cached`, if you want.
(Since this is a reduction, where `Mir` was replaced with `String`)

Reply:
Well received. Renamed to `Cached` then !